src/main/java/bankProject/service/BankService.java

```diff
@@ -1,24 +1,200 @@
 package bankProject.service;
 
+import bankProject.export.SummaryStatistics;
 import bankProject.model.BankTransaction;
-import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Month;
+import java.util.*;
+import java.util.stream.Collectors;
 
 public class BankService {
 
     private List<BankTransaction> bankTransactions;
+    private final Logger log = LoggerFactory.getLogger(getClass());
 
     public BankService(List<BankTransaction> bankTransactions) {
-        this.bankTransactions = bankTransactions;
+         this.bankTransactions = bankTransactions;
+    }
+
+    // 총 입금, 출금 내역
+    public int totalAmount() {
+        int total = bankTransactions.stream()
+                                     .mapToInt(BankTransaction::getAmount)
+                                     .sum();
+
+        log.info(String.valueOf(total));
+
+        return total;
+    }
+
+    // 총 수입 내역
+    public int incomeAmount() {
+        int income = bankTransactions.stream()
+                                     .filter(amount -> amount.getAmount() > 0)
+                                     .mapToInt(BankTransaction::getAmount)
+                                     .sum();
+
+        log.info(String.valueOf(income));
+
+        return income;
+    }
+
+    // 총 지출 내역
+    public int expenseAmount() {
+        int expense = bankTransactions.stream()
+                .filter(amount -> amount.getAmount() < 0)
+                .mapToInt(BankTransaction::getAmount)
+                .sum();
+
+        log.info(String.valueOf(expense));
+
+        return expense;
+    }
+
+    // 각 개월마다의 총합
+    public int monthlyAmount(Month month) {
+
+        if (month == null) {
+            throw new NullPointerException("month should not be null");
+        }
+
+        int monthlyTotal = bankTransactions.stream()
+                                            .filter(monthlyAmount -> monthlyAmount.getDate().getMonth() == month)
+                                            .mapToInt(BankTransaction::getAmount)
+                                            .sum();
```

Comment:
int monthlyTotal2 = bankTransactions.stream()
                .filter(amountMonthly -> amountMonthly.getDate().getMonth() == month)
                        .mapToInt(amountMonthly -> {
                            Integer amount = amountMonthly.getAmount();
                            if (amount == null) {
                                log.warn("Transaction amount is null " +amountMonthly.toString());
                            }
                            return amount;
                        }).sum();